.github/workflows/tests.yaml

```diff
@@ -95,40 +122,38 @@ jobs:
       - SwiftBuild
     steps:
     - uses: actions/checkout@master
+    - name: Download build artifacts
+      uses: actions/download-artifact@v3
+      with:
+        path: ../iOS-Build.tgz
+        name: iOS-Build
+    - name: Prepare build artifacts
+      run: |
+        tar -xzf ../iOS-Build.tgz
     - uses: ./.github/actions/setup-secrets
       with:
         okta_plist: "${{ secrets.TEST_OKTA_PLIST }}"
         test_configuration: "${{ secrets.TEST_CONFIGURATION }}"
     - name: Test Web Sign In
       run: |
         xcodebuild \
-            -derivedDataPath ~/Build/DerivedData \
-            -clonedSourcePackagesDirPath ~/Build/ClonedSources \
-            -resultBundlePath ~/TestResults/WebSignIn-iOS.xcresult \
```

Comment:
Why was ```-resultBundlePath ~/TestResults/WebSignIn-iOS.xcresult``` removed?

Reply:
Exporting the result bundle is only necessary if you want to post-process the test results, which includes screen shots of every step of the UI tests.